server/src-lib/Hasura/Events/Lib.hs

```diff
@@ -166,49 +168,86 @@ retryAfterHeader = "Retry-After"
 
 initEventEngineCtx :: Int -> DiffTime -> STM EventEngineCtx
 initEventEngineCtx maxT fetchI = do
-  q <- TQ.newTQueue
-  c <- newTVar 0
-  return $ EventEngineCtx q c maxT fetchI
-
-forkEventQueueProcessors
+  c <- newTVar maxT
```

Comment:
I know the naming here were screwed up to begin with but maybe we can few of them here like `c`